### PR TITLE
@damassi => Pass mediator to Reaction apps properly

### PR DESCRIPTION
**Pull request by mzikherman**
I think this is the way we're supposed to pass these around now. Currently in prod the artist page isn't grabbing the mediator, so it's leading to some bugs (modals, etc.).

Locally this fixes it.

### CODE REVIEW

**Comment by sweir27** [comment]
Ah damn, gotta do this for the order app too.

**Comment by mzikherman** [comment]
Yea, I updated that one here.

File: src/desktop/apps/artist2/client.js [hunk 1]
```diff
@@ -20,12 +20,13 @@ buildClientApp({
   routes,
   context: {
     user: sd.CURRENT_USER,
+    mediator,
```
Comment (damassi) [inline src/desktop/apps/artist2/client.js]: Oof 😦

Comment (damassi) [reply src/desktop/apps/artist2/client.js]: Refactored everything for this specific reason and then didn't update the actual app.

**Comment by damassi** [comment]
👍 thanks for fixing

merge on green

File: src/desktop/apps/order2/client.js [hunk 2]
```diff
@@ -47,7 +48,7 @@ buildClientApp({
   .then(({ ClientApp }) => {
     ReactDOM.hydrate(
       <Container>
-        <ClientApp mediator={mediator} />
+        <ClientApp />
```
Comment (sweir27) [inline src/desktop/apps/order2/client.js]: Oh! Thanks @mzikherman 😄

**Pull request merged**